packages/manager/src/features/LoadBalancers/LoadBalancerDetail/LoadBalancerServiceTargets.tsx

```diff
@@ -50,6 +58,14 @@ export const LoadBalancerServiceTargets = () => {
     ['+order_by']: orderBy,
   };
 
+  const handleDeleteServiceTarget = (serviceTarget: ServiceTarget) => {
+    if (serviceTarget) {
+      setIsDeleteDialogOpen(true);
+    }
+
+    setSelectedServiceTarget(serviceTarget);
```

Comment:
Do we need an `if` here?

```suggestion
    setIsDeleteDialogOpen(true);
    setSelectedServiceTarget(serviceTarget);
```